configure

```diff
@@ -1366,7 +1366,10 @@ if (exists $opts{'gtest'} || $tests) {
       $gtest_root = File::Spec->catdir($sm_dir, 'build', 'install');
       $build_gtest = ! -d $gtest_root;
       $opts{'gtest'} = $sm_dir;
-      # No need to set env. variable with submodule.
+      # No need to set env. variable with submodule UNLESS we're in safetyprofile
+      if ($opts{'safety-profile'}) {
+        setEnv('GTEST_ROOT', $gtest_root);
```

Comment:
Should this have a log message when verbose is on? Or is it already handled somewhere else?